src/esm_parser/esm_parser.py

```diff
@@ -718,12 +771,21 @@ def deep_update(chapter, entries, config, blackdict={}):
             # Trying to update a dictionary from the same file (in blackdict)
             # an ``add_`` returns an error
             if isinstance(blackdict[chapter], dict):
+                # Keys don't have provenance. In this case we know we are dealing with a
+                # choose block so the line of the conflicting choose is the line of the
+                # first value -3.
```

Comment:
What about whitespace?

Reply:
That is a very good point, however, I could not figure out how to do this without implementing provenance on keys. Feel free to open an issue for that, we can solve it at another time.